docs/locale/fr/LC_MESSAGES/userguide/recovery_device.po

```diff
@@ -0,0 +1,207 @@
+
+#: ../../userguide/recovery_device.rst:13
+msgid "Open the ``Recovery device`` menu."
+msgstr "Ouvrir le menu ``Récupération d'appareils``."
+
+#: ../../userguide/recovery_device.rst:15
+msgid ""
+"You need to open the sub-menu then click on ``Recovery device`` option "
+"(alternatively, you can use the shortcut ``Ctrl+I``)."
+msgstr ""
+"Vous avez besoin d'ouvrir le sous-menu puis cliquer sur ``récupération "
+"d'appareil`` (Alternativement, vous pouvez utiliser le raccourci ``Ctrl+I``)."
+
+#: ../../userguide/recovery_device.rst
+#, fuzzy
+msgid "Parsec menu with recovery device option highlighted"
+msgstr "Le menu Parsec avec l'option récupération d'appareil surligné"
+
+#: ../../userguide/recovery_device.rst:20
+msgid "That will open a modal dialog that looks like this:"
+msgstr "Ça va ouvrir un dialogue qui ressemble à ceci:"
+
+#: ../../userguide/recovery_device.rst
+#, fuzzy
+msgid "Parsec recovery device modal"
+msgstr "Dialogue Parsec de récupération d'appareil"
+
+#: ../../userguide/recovery_device.rst:26
+msgid "Create a new recovery device"
+msgstr "Créer un nouvel appareil de récupération"
+
+#: ../../userguide/recovery_device.rst:28
+msgid "In the Device recovery dialog, select ``Create a recovery device``"
+msgstr ""
+"Dans le dialogue de récupération d'appareil, sélectionner ``Créer un "
+"appareil de récupération``"
+
+#: ../../userguide/recovery_device.rst:30
+#: ../../userguide/recovery_device.rst:61
+msgid "From here:"
+msgstr "À partir de là:"
+
+#: ../../userguide/recovery_device.rst:32
+msgid "Select the device you would like to create a recovery device for."
+msgstr ""
+"Sélectionner l'appareil à partir duquel vous voulez créer un appareil de "
+"récupération."
+
+#: ../../userguide/recovery_device.rst:33
+msgid ""
+"Click the ``Select`` button to specify a file to store the recovery device."
+msgstr ""
+"Cliquer sur le bouton ``Choisir`` pour spécifier le fichier où sera "
+"sauvegardé l'appareil de récupération."
+
+#: ../../userguide/recovery_device.rst
+#, fuzzy
+msgid "Create a recovery device modal"
+msgstr "Dialogue de création d'un appareil de récupération"
+
+#: ../../userguide/recovery_device.rst:38
+msgid ""
+"Click ``Next``, you will be prompted to enter the password for the device "
+"you have selected"
+msgstr ""
+"Cliquer sur ``Suivant``, vous serez demandé d'entrer le mot de passe pour "
+"l'appareil que vous avez sélectionné"
+
+#: ../../userguide/recovery_device.rst
+#, fuzzy
+msgid "Create a recovery device re-authenticate modal"
+msgstr ""
+"Dialogue de reconnection pour la création d'un appareil de récupération"
+
+#: ../../userguide/recovery_device.rst:43
+msgid ""
+"After confirmation, the recovery file is created and a passphrase is "
+"displayed. **You need to save the passphrase** as it will not be displayed "
+"again."
+msgstr ""
+"Après confirmation, le fichier de récupération est créé et la phrase de "
+"passe est affichée. **Vous avez besoin de sauvegarder la phrase de passe** "
```

Comment:
<a href="https://imgflip.com/i/85ubn3"><img src="https://i.imgflip.com/85ubn3.jpg" title="made at imgflip.com"/></a><div><a href="https://imgflip.com/memegenerator">from Imgflip Meme Generator</a></div>

Reply:
The glossary does not state the translation to use for `passphrase`.

Linguee suggest (https://www.linguee.fr/francais-anglais/search?source=auto&query=passphrase):

- `phrase secrete`
- `phrase de passe`

Reply:
"Frappez la phrase de passe" fait un joli virelangue :sweat_smile: 

"phrase secrete" sounds much better :pray:

Reply:
done